RELEASE_PROCESS.md

```diff
@@ -0,0 +1,38 @@
+# Release Process For AsyncAPI Specification
```

Comment:
```suggestion
# Release Process Of The AsyncAPI Specification
```

I want to change because in below paragraph you use the `... of the ...` sentence :)